packages/core/src/routes/experience/classes/verifications/verification.ts

```diff
@@ -0,0 +1,17 @@
+import type { VerificationType } from '@logto/schemas';
+
+/**
+ * Parent class for all verification records
+ */
+export abstract class Verification {
+  abstract readonly id: string;
+  abstract readonly type: VerificationType;
```

Comment:
will it be better to make it a generic class? e.g.

```ts
class Verification<T extends VerificationType> {
   abstract readonly type: T;
}
```

Reply:
This might cause extra efforts to handle the type of `verificationRecords` list in the interaction.